### PR TITLE
Update Team listing to reflect recent additions/retirements

### PR DESCRIPTION
**Pull request by Ephemeralis**
Adds the following people to the listing:

* cYsmix (sound design & FA)
* nekodex (osu!web developer & FA)
* Shiro (player support)

Moves the following to inactive:

* ztrot
* Loctav (retired)

### CODE REVIEW

**Comment by Xinnoh** [comment]
nekodex isn't on the FA listing 🤔

**Comment by Ephemeralis** [comment]
Soon(tm)

File: wiki/People/The_Team/en.md [hunk 1]
```diff
@@ -13,15 +13,16 @@ Active
...
 | [flyte](https://osu.ppy.sh/u/3103765), [osu!next](https://next.ppy.sh/) | Lead designer |
-| [Loctav](https://osu.ppy.sh/u/71366) | Official tournament organization, Staff and community management |
-| [nanaya](https://osu.ppy.sh/u/2387883) | Web implementation |
+| [nanaya](https://osu.ppy.sh/u/2387883) | osu!web developer |
+| [nekodex](https://osu.ppy.sh/u/104) | osu!web developer & official osu! Featured Artist |
 | [RBRat3](https://osu.ppy.sh/u/307202) | osu!painter, Cool stuff maker-guy extraordinaire |
 | [smoogipooo](https://osu.ppy.sh/u/1040328) | osu!mania development, Bug busting |
+| [Shiro](https://osu.ppy.sh/u/113005) | Player support |
 | [Tom94](https://osu.ppy.sh/u/1857058) | pp development |
 | [yelle](https://osu.ppy.sh/u/4916903) | [osu!store](/wiki/osu!store) management |
```
Comment (MegaApplePi) [inline wiki/People/The_Team/en.md]: can you remove this wiki link? (or relink to <https://osu.ppy.sh/store/listing>)

**Comment by Ephemeralis** [comment]
Outdated and fixed the store link.

File: wiki/People/The_Team/en.md [hunk 2]
```diff
@@ -13,15 +13,16 @@ Active
...
 | ---- | ----------- |
-| [Ephemeral](https://osu.ppy.sh/u/102335) | Player support, Meganekko secretary |
-| [flyte](https://osu.ppy.sh/u/3103765), [osu!next](https://next.ppy.sh/) | Lead designer |
-| [Loctav](https://osu.ppy.sh/u/71366) | Official tournament organization, Staff and community management |
-| [nanaya](https://osu.ppy.sh/u/2387883) | Web implementation |
+| [cYsmix](https://osu.ppy.sh/u/272870) | Audio design & official osu! Featured Artist |
+| [Ephemeral](https://osu.ppy.sh/u/102335) | Media, Featured Artist outreach, Meganekko secretary |
+| [flyte](https://osu.ppy.sh/u/3103765) | Lead designer |
+| [nanaya](https://osu.ppy.sh/u/2387883) | osu!web developer |
+| [nekodex](https://osu.ppy.sh/u/104) | osu!web developer & official osu! Featured Artist |
```
Comment (cl8n) [inline wiki/People/The_Team/en.md]: nekodex is /u/102

File: wiki/People/The_Team/en.md [hunk 3]
```diff
@@ -1,46 +1,51 @@
...
-| [ztrot](https://osu.ppy.sh/u/6347) | [osu!academy](/wiki/Announcements/osu!academy) professor, Media/character design |
-| [Zallius](https://osu.ppy.sh/u/55) | Endangered species |
+| [![][flag_NO] cYsmix](https://osu.ppy.sh/u/272870) | Audio design & [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| [![][flag_AU] Ephemeral](https://osu.ppy.sh/u/102335) | Media, Featured Artist outreach, Meganekko secretary |
+| [![][flag_JP] flyte](https://osu.ppy.sh/u/3103765) | Lead designer, [osu!next](https://next.ppy.sh/) |
+| [![][flag_JP] nanaya](https://osu.ppy.sh/u/2387883) | osu!web developer |
+| [![][flag_AU] nekodex](https://osu.ppy.sh/u/102) | osu!web developer |
+| [![][flag_US] RBRat3](https://osu.ppy.sh/u/307202) | osu!painter, Cool stuff maker-guy extraordinaire |
+| [![][flag_AU] smoogipooo](https://osu.ppy.sh/u/1040328) | osu!mania development, Bug busting |
+| [![][flag_FR] Shiro](https://osu.ppy.sh/u/113005) | Player support |
```
Comment (MegaApplePi) [inline wiki/People/The_Team/en.md]: can you move this one up (to alphabetize)?

**Pull request merged**